API/ExecutorInterface.php

```diff
@@ -27,4 +27,13 @@ public function supportedTypes();
      * @throws \Exception
      */
     public function execute(MigrationStep $step);
+
+    /**
+     * Generate a update template
+     *
+     * @param string $identifier
+     * @return string Yaml dump
+     * @throws \Exception
+     */
+    public function generateTemplate($identifier);
```

Comment:
PS: on a side note: adding a method to an interface generally requires a major version bump...
...it is true that in this case you provided for an implementation in the base (abstract) class. I will have to think it through  a bit...